udf/hmac_sha256.sql

```diff
@@ -0,0 +1,51 @@
+CREATE OR REPLACE FUNCTION udf.hmac_sha256(key BYTES, message BYTES) AS (
```

Comment:
I have validated this against the `crypto` JS library they were using [here](https://github.com/mozilla/fxa-amplitude-send/blob/89bfaef20a1d978fce2dccddc0155699a17ac172/pubsub/utils.js#L12), just to ensure everything matches.

Reply:
Well, this is a game changer. I did not expect we'd be able to implement this as SQL. Can you add a reference here for where this algorithm comes from?

Reply:
Luckily `BYTES` support is pretty strong in BQ.